tests/integration/golang/database/import_test.go

```diff
@@ -0,0 +1,246 @@
+//go:build integration
+
+package database
```

Comment:
since we are going more tool I think, can we organise tests under the different path -> `tests/integration/golang/tools|console/database/import_test.go`

Reply:
I think we should match the pkg path